src/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.cs

```diff
@@ -74,109 +74,109 @@ public enum CompareMethod
         [System.ObsoleteAttribute("For a carriage return and line feed, use vbCrLf.  For the current platform's newline, use System.Environment.NewLine.")]
-        public const VbStrConv vbNarrow = VbStrConv.Narrow;
+        public const Microsoft.VisualBasic.VbStrConv vbNarrow = Microsoft.VisualBasic.VbStrConv.Narrow;
         public const string vbNewLine = "\r\n";
-        public const MsgBoxResult vbNo = MsgBoxResult.No;
-        public const FileAttribute vbNormal = FileAttribute.Normal;
-        public const AppWinStyle vbNormalFocus = AppWinStyle.NormalFocus;
-        public const AppWinStyle vbNormalNoFocus = AppWinStyle.NormalNoFocus;
-        public const VariantType vbNull = VariantType.Null;
+        public const Microsoft.VisualBasic.MsgBoxResult vbNo = Microsoft.VisualBasic.MsgBoxResult.No;
+        public const Microsoft.VisualBasic.FileAttribute vbNormal = Microsoft.VisualBasic.FileAttribute.Normal;
+        public const Microsoft.VisualBasic.AppWinStyle vbNormalFocus = Microsoft.VisualBasic.AppWinStyle.NormalFocus;
+        public const Microsoft.VisualBasic.AppWinStyle vbNormalNoFocus = Microsoft.VisualBasic.AppWinStyle.NormalNoFocus;
+        public const Microsoft.VisualBasic.VariantType vbNull = Microsoft.VisualBasic.VariantType.Null;
         public const string vbNullChar = "\0";
         public const string vbNullString = null;
-        public const VariantType vbObject = VariantType.Object;
-        public const int vbObjectError = unchecked((int)0x80040000);
-        public const MsgBoxResult vbOK = MsgBoxResult.Ok;
-        public const MsgBoxStyle vbOKCancel = MsgBoxStyle.OkCancel;
-        public const MsgBoxStyle vbOKOnly = MsgBoxStyle.ApplicationModal;
-        public const VbStrConv vbProperCase = VbStrConv.ProperCase;
-        public const MsgBoxStyle vbQuestion = MsgBoxStyle.Question;
-        public const FileAttribute vbReadOnly = FileAttribute.ReadOnly;
-        public const MsgBoxResult vbRetry = MsgBoxResult.Retry;
-        public const MsgBoxStyle vbRetryCancel = MsgBoxStyle.RetryCancel;
-        public const FirstDayOfWeek vbSaturday = FirstDayOfWeek.Saturday;
-        public const CallType vbSet = CallType.Set;
-        public const DateFormat vbShortDate = DateFormat.ShortDate;
-        public const DateFormat vbShortTime = DateFormat.ShortTime;
-        public const VbStrConv vbSimplifiedChinese = VbStrConv.SimplifiedChinese;
-        public const VariantType vbSingle = VariantType.Single;
-        public const VariantType vbString = VariantType.String;
-        public const FirstDayOfWeek vbSunday = FirstDayOfWeek.Sunday;
-        public const FileAttribute vbSystem = FileAttribute.System;
-        public const MsgBoxStyle vbSystemModal = MsgBoxStyle.SystemModal;
+        public const Microsoft.VisualBasic.VariantType vbObject = Microsoft.VisualBasic.VariantType.Object;
+        public const int vbObjectError = -2147221504;
```

Comment:
cc @cston, @333fred to spot check some of these changes. From my perspective, most of these normalizations are correct and cosmetic (there should be no observable change from the callers view).